crates/debuginfo/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "miden-debuginfo"
```

Comment:
nit (feel free to ignore): would `miden-debug-types` be a more appropriate name?

Reply:
That seems reasonable to me, `miden-debuginfo` was all I could come up with that seemed reasonable, but I like `miden-debug-types`